platform/solr/solr-factory/src/main/java/org/codice/solr/factory/SolrServerHttpRequestRetryHandler.java

```diff
@@ -29,30 +29,37 @@
 
 public class SolrServerHttpRequestRetryHandler implements HttpRequestRetryHandler {
 
-    private static final Integer MAX_RETRY_COUNT = 11;
+    private static final int MAX_ATTEMPTS = 10;
 
     private static final Logger LOGGER =
             LoggerFactory.getLogger(SolrServerHttpRequestRetryHandler.class);
 
     @Override
-    public boolean retryRequest(IOException e, int retryCount, HttpContext httpContext) {
+    public boolean retryRequest(IOException e, int executionCount, HttpContext httpContext) {
         if (e instanceof InterruptedIOException) {
-            LOGGER.error("Connection timeout.");
-        }
-        if (e instanceof UnknownHostException) {
-            LOGGER.error("Unknown host.");
-        }
-        if (e instanceof SSLException) {
-            LOGGER.error("SSL handshake exception.");
+            LOGGER.error("Connection timeout.", e);
+        } else if (e instanceof UnknownHostException) {
+            LOGGER.error("Unknown host.", e);
+        } else if (e instanceof SSLException) {
+            LOGGER.error("SSL handshake exception.", e);
         }
+
         HttpClientContext clientContext = HttpClientContext.adapt(httpContext);
         HttpRequest request = clientContext.getRequest();
         if (!(request instanceof HttpEntityEnclosingRequest)) {
             LOGGER.error("Connection failed. Request is idempotent.");
         }
+
+        if (executionCount == MAX_ATTEMPTS) {
+            LOGGER.error(
+                    "The maximum number of attempts ({}) has been reached. No more attempts will be made.",
+                    MAX_ATTEMPTS);
```

Comment:
Are there steps to force Solr to reconnect?  Should we state something about restarting the solr bundle or something to force a retry?